config/checkstyle/space-clean

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+git status | grep java | awk '{print $3}' | xargs -o -n 1 vim -c ":retab" -c ":%s/\s\+$//g" -c ":wq" > /dev/null
```

Comment:
I assume this is supposed to be run before committing.
I'm not sure if you really care about this, but this command fails when there are no java file changes, because the arg to vim is empty.